iphone/Maps/UI/BottomMenu/TabBar/BottomTabBarViewController.swift

```diff
@@ -14,6 +14,7 @@ class BottomTabBarViewController: UIViewController {
   private var avaliableArea = CGRect.zero
   @objc var isHidden: Bool = false {
     didSet {
+      view.isUserInteractionEnabled = !isHidden
```

Comment:
Is this line required considering other changes in the switch above?

Reply:
Generally, no